rendered/manifests/tests/splunk-connect-for-snmp/templates/inventory/job.yaml

```diff
@@ -44,6 +44,8 @@ spec:
             value: "http://release-name-mibserver/standard.txt"
           - name: LOG_LEVEL
             value: INFO
+          - name: CHAIN_OF_TASKS_EXPIRY_TIME
+            value: "60"
```

Comment:
Is this variable related to walk max retry?

Reply:
no, this is from this PR: https://github.com/splunk/splunk-connect-for-snmp/pull/838 , no `make render` was executed there